Add instructions to install legacy fatiando (#47)
fatiando/website: install/index.md

[anaconda]: https://www.anaconda.com/products/individual
[swc-install]: https://carpentries.github.io/workshop-template/#python

---

> # Installing the legacy `fatiando` package

Reviewer: ```suggestion
## Installing the legacy `fatiando` package
```

```{warning}

> The old `fatiando` package **is not currently maintained nor developed**, it

Reviewer: ```suggestion
The old `fatiando` package **is not currently maintained or developed**, it
```

> means that **it won't get any bugfix or any new feature**. The purpose of these

Reviewer: ```suggestion
meaning that **it won't get bugfixs or new feature**. The purpose of these
```

instructions are for **legacy use only**, in order to run old code to check
reproducibility of results.
```

```{tip}
If you are working on some new research or data analysis, we strongly

> **recommend to use the [latest tools](#install-the-latest-tools)**.

Reviewer: ```suggestion
**recommend using the [latest tools](#install-the-latest-tools)** instead.
```

```

If you need to install the old `fatiando` package, you can still do so through

> `conda`, therefore you need [Anaconda][anaconda] to be installed.

Reviewer: ```suggestion
`conda`.
```

Remove this part since we should probably start recommend Miniforge instead.

Since the old `fatiando` runs on Python 2.7, a deprecated version of Python,

> you need to create a separated environment for our installation in order to

Reviewer: ```suggestion
you need to create a separate environment for our installation in order to
```

avoid conflicts with your Anaconda installation.

Create a new environment and install the required packages:

```
conda create --name fatiando-legacy -c conda-forge python=2.7 numpy=1.10 scipy=1.2 matplotlib=2.2 pillow=6.2 future=0.18 numba=0.43 pytest=4.6 fatiando=0.5
```

Activate the newly created environment:

```bash
conda activate fatiando-legacy
```

And test your installation:

```bash
python -c "import fatiando; fatiando.test()"
```

You should see a green dot after each test is run and no red `F`.

> The documentation for the legacy `fatiando` package is available in

Reviewer: ```suggestion
The documentation for the legacy `fatiando` package is available at
```
